test/tests-no-moment.js

```diff
@@ -42,23 +42,6 @@ define(function(require){
 		};
 	});
 
-	require('test/checkbox-test');
-	require('test/combobox-test');
 	require('test/datepicker-test');
```

Comment:
I've added the `?no-moment=true` URL parameter to the `test` and `releasetest` tasks.